File: 268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int Total_Sum=n*(n+1)/2;
        int Partial_Sum=0;
        for(int i=0;i<n;i++)
        {
            Partial_Sum=Partial_Sum+nums[i];
        }
        return Total_Sum-Partial_Sum;
        
    }
};

File: 268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n=len(nums)
        Total_Sum=n*(n+1)//2
        Partial_Sum=0
        for i in range(n):
            Partial_Sum=Partial_Sum+nums[i]
        return Total_Sum-Partial_Sum